eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -121,6 +121,8 @@
   <!-- Suppress warnings for Event Processor until the usage of "Client" is discussed and resolved: https://github.com/Azure/azure-sdk/issues/321 -->
   <suppress checks="com.azure.tools.checkstyle.checks.ServiceClientBuilderCheck" files=".*EventHubClientBuilder.java"/>
 
+  <suppress checks="com.azure.tools.checkstyle.checks.ServiceClientBuilderCheck" files="com.azure.core.http.netty.NettyAsyncHttpClientBuilder.java"/>
```

Comment:
@mssfang The checkstyle rule here is requiring that I suppress this issue, because it requires that all builders have the `@ServiceClientBuilder` annotation. We should find a way to make this check more specific or less demanding, as there will be cases such as this where it is a builder but not a service client. Can you please give it some thought and get back to me?